parachain/runtime/testnet-kintsugi/src/lib.rs

```diff
@@ -87,8 +87,8 @@ impl_opaque_keys! {
 /// This runtime version.
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-    spec_name: create_runtime_str!("testnet-parachain"),
-    impl_name: create_runtime_str!("testnet-parachain"),
+    spec_name: create_runtime_str!("testnet-kintsugi"),
+    impl_name: create_runtime_str!("testnet-kintsugi"),
```

Comment:
This'll break clients - should we make them forward compatible with this change?

Reply:
See: https://github.com/interlay/interbtc-clients/pull/318